ansible/inventory.yml

```diff
@@ -324,9 +324,16 @@ hosts:
         win2012r2_vs2015-x64-2: {ip: 104.130.141.137}
         win2012r2_vs2019-x64-1: {ip: 162.242.237.124}
         win2012r2_vs2019-x64-2: {ip: 104.130.158.58}
-        win2012r2_vs2019-x64-3: {ip: 119.9.131.63}
-        win2012r2_vs2019-x64-4: {ip: 104.130.219.103}
-        win2012r2_vs2019-x64-6: {ip: 104.130.141.231}
+        win2019_vs2019-x64-1: {}
+        win2019_vs2019-x64-2: {}
+        win2019_vs2019-x64-3: {}
+        win2019_vs2019-x64-4: {}
+        win2022_vs2022-x64-1: {}
+        win2022_vs2022-x64-2: {}
+        win2022_vs2022-x64-3: {}
+        win2022_vs2022-x64-4: {}
+        win2022_vs2022-x64-5: {}
+        win2022_vs2022-x64-6: {}
```

Comment:
Aren't these missing the IP address?

Reply:
I planned to add them to the secrets repo as an additional safety measurement, but if they should be here for consistency with other machines or any other reason, I'll add them.

Reply:
Adding them to the secrets repo is okay -- as long as they're added somewhere 🙂.